database-postgres/data-generator/generate-publishers.js

```diff
@@ -0,0 +1,29 @@
+const faker = require('faker');
+const fs = require('fs');
+const {writeData} = require('../writeToCSV.js');
+
+// Include process module 
+const process = require('process'); 
+
+/* 
+* @function generatePublishers
+*
+* @typedef {object} Data
+* @property {string} publisher - name of publisher
+*
+* @returns {Data} one fake data record
+*/
+
+
+/* Generates 1 publisher record */
+const generatePublishers = () => {
+ 
+  //one row of data
+  let record = {publisher: faker.lorem.words()};
+  return record;
+};
+
+module.exports.generatePublishers = generatePublishers;
+
+//write products data to csv if ran from command line
+process.argv[2] === '--generate' ? writeData(5e6, generatePublishers, 'publishers') : null;
```

Comment:
What happens if someone runs your command using `yarn` instead of `npm run`? It's likely that `argv[2]` won't be what is expected here. Just food for thought!